test/schema_test.go

```diff
@@ -40,17 +38,20 @@ type testCase struct {
 	manifest  string
 }
 
-var mutatedYamlContentMap = map[string]string{}
-var mutationTestCasesMap = map[string][]testCase{}
-
-func init() {
```

Comment:
This change is not required, I'm just `init()` averse, but I don't mind reverting it

Reply:
I don't have strong opinions for a test file, but I tend to agree that the init() pattern leads to more confusion than it helps